src/System.Private.ServiceModel/tools/CertificateCleanup/Program.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace System.Private.ServiceModel.Tests.Tools
+{
+    class Program
+    {
+        private const string ClientCertificateSubject = "WCF Client Certificate";
+        private const string CertificateIssuer = "DO_NOT_TRUST_WcfBridgeRootCA";
+
+        static bool s_debug = false; 
```

Comment:
Nit: I'd favor either all cmd line arguments having static fields, or creating a small data class that holds them that you pass to the worker methods.  Not critical, just stylistic.